src/xenserver/opt/xensource/www/XCP-ng-index.html

```diff
@@ -286,6 +289,96 @@
       .support p {
         font-weight: 600;
       }
```

Comment:
I think the font defined from line 166 to 291 aren't used anymore. That would allo to remove the font files

Reply:
You mean 166 to 209, the others css code is not about the font.

Reply:
I removed it too